ui/design-system-core/src/components/Avatar/index.tsx

```diff
@@ -91,7 +93,7 @@ const Avatar: React.FC<IAvatarProps> = props => {
     'bg-grey6 dark:bg-grey6 opacity-25 z-10 absolute top-0 left-0 w-full h-full';
 
   return (
-    <div className={tw(className)} onClick={onClick}>
+    <div className={tw(`${containerStyle} ${className}`)} onClick={onClick}>
```

Comment:
Can you add a fallback for the className here, like `${className ?? ''}` else when it is not defined, it will add `undefined` as class